server/lib/adapters/validate-result-columns.js

```diff
@@ -0,0 +1,21 @@
+import _ from "lodash";
+
+export default function validate(columnNames) {
+  const errors = [];
+
+  if (!_.includes(columnNames, "email") && !_.includes(columnNames, "external_id")) {
+    errors.push("Column names should include at least one required parameters: email or external_id");
```

Comment:
"Column names should include at least one required parameters: email or external_id."
-> 
"Column names should include email and/or external_id"